gettsim/incomes.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+import pandas as pd
+
+from gettsim.func_out_columns import DPI
+from gettsim.func_out_columns import GROSS
+
+
+def disposible_income(df):
+    disp_inc = pd.DataFrame(index=df.index, columns=DPI)
+    disp_inc["dpi_ind"] = df[
+        [
+            "m_wage",
+            "m_kapinc",
+            "m_self",
+            "m_vermiet",
+            # "m_imputedrent", Do we need this?
+            "m_pensions",
+            "m_transfers",
+            "kindergeld",
+            "uhv",
+        ]
+    ].sum(axis=1) - df[
+        ["incometax", "soli", "abgst", "gkvbeit", "rvbeit", "pvbeit", "avbeit"]
+    ].sum(
+        axis=1
+    )
+
+    # Disposible income on hh level
+    # TODO: Why we do not have m_alg2, wohngeld and kiz
```

Comment:
because you need to run through `benefit_checks()` first. Before, this was done within the `kiz` function. We haven't touched `tax_transfer()` for quite a while, so this was not updated.

Reply:
I included it, but commented out the assert checks. Can we agree on having a general checking section, if the inputs are valid? Then we would get valid outcomes.

Reply:
we can outsource the validation checks. The first one (`assert bp[ben].notna().all()`) could be dropped completely. It was mainly important when writing the function.